azure/src/main/java/com/microsoft/azure/implementation/Azure.java

```diff
@@ -211,11 +210,6 @@ public Azure withDefaultSubscription() throws CloudException, IOException {
     }
 
     public interface ResourceGroup extends com.microsoft.azure.management.resources.ResourceGroup {
```

Comment:
yes, I see what you mean. there may be some "empty shell" definitions I may have left behind.